### ISSUE TITLE
Installed "templates-all_2.2-1.deb" don't show up in Templete Manager as explained in Install-Guide.

### ISSUE THREAD

**Issue by PranavBhattarai** [dohliam/libreoffice-impress-templates#16]
Distro: Ubuntu 19.10
LO: Version: 6.4.1.2
I followed the **[installation guide](https://github.com/dohliam/libreoffice-impress-templates/wiki/Ubuntu-Debian-Install-Guide)** to install all template using 
`sudo dpkg -i libreoffice-impress-templates-all_2.2-1.deb` command, as u can see:

![image](https://user-images.githubusercontent.com/32763112/76696229-2ee18e80-66b1-11ea-90ac-1989f280bba1.png)

The only templates I see is the one which came by default, as u can see:
![Screencast from 15-03-20 11_37_45 AM +0545](https://user-images.githubusercontent.com/32763112/76696274-b4653e80-66b1-11ea-9ca9-45e5eb93c932.gif)

_I think the directory which .deb is installing slides to, is the old one. Not the new one._

**Comment by dohliam** [comment]
Thanks very much for raising this issue! I think you're right that it's an issue with the location of the template files, and the deb package is overdue for an update in any case.

From a quick check, it appears that templates are currently stored in `/usr/lib/libreoffice/share/template/common/presnt/`. Can you confirm that this is where they are located on your system? Assuming it is, it should be fairly simple to rebuild the package.

**Comment by dohliam** [comment]
Just realized/remembered that this may be dependent on your system's locale settings -- so the `common` part of the directory path could be something else.

I'm not sure if there is an easy way to customize the install destination based on the OS default locale (though if there are any Debian packaging gurus reading this please feel free to chime in!). So the easiest thing to do for now might be to just download the regular release zip file with all the templates and manually copy it to your Libreoffice templates directory. In other words, something like this: `sudo cp *.otp /usr/lib/libreoffice/share/template/[locale]/presnt`, with `[locale]` replaced by your locale.

**Comment by PranavBhattarai** [comment]
@dohliam yes the templates are installed in u mentioned:
![Screenshot from 2020-03-15 18-28-35](https://user-images.githubusercontent.com/32763112/76701617-d4652400-66ea-11ea-85b8-3c13b5b37a31.png)

Note: I have installed LibreOffice using **Flatpak**. Does this change the installation directory of where the LibreOffice resides? Then I might need to move all the templates to that directory.

**Comment by dohliam** [comment]
@PranavBhattarai that's a possibility, though not sure how Flatpak handles directory structure. It seems from your screenshot that you have both the default LibreOffice templates (for example, `Alizarin.otp` and `Blue_Curve.otp`) and the additional templates (`alizarin.otp` and `blue_curve.otp` -- note the lowercase filenames). So they seem to be in the right place, unless you have more than one installation of LibreOffice (perhaps a default one that came with your system as well as the one you installed with Flatpak?) at the same time. The only other thing I can think of would be to try closing and restarting LibreOffice and see if the templates appear after that.

**Comment by PranavBhattarai** [comment]
@dohliam 
No, I don't have two LibreOffice installed. I uninstall LO installed by both **apt** & **snap** repo before I install LO from **Flatpak**.
No, matter how much I restart LO, it won't show me the installed templates.

I didn't see that coming. The existence of duplicates files is really started bothering me. 
I think I should create a GitHub issue in Flapak repo linking this issue.

**Comment by dohliam** [comment]
@PranavBhattarai The duplicate files are not actually the fault of Flatpak. They are in fact the same templates, but since they are included by default in LibreOffice (with uppercase filenames) and also this repo (with lowercase filenames), they appear twice. The duplicates can be fairly easily removed though from the LO template interface. The question now is just why LO is not picking up on the existence of the templates in its own template folder.

**Comment by MyPyDavid** [comment]
I had this same issue at first. However, after clicking the  '**Refresh**' button in the settings (bottom left) of the Templates manager window, all of the templates appeared. 
I did the Graphical Install, all the files were also installed to `/usr/lib/libreoffice/share/template/common/presnt/` and the default files are also there.
Thanks for this package! @dohliam
